### ISSUE TITLE
Graphql not working? Error when parsing json

### ISSUE THREAD

**Issue by jaroslaw-weber** [mgattozzi/github-rs#140]
So I have this function:
```
pub fn get_graph_ql_request() -> String
{
r#"
{
repositoryOwner(login: "jaroslaw-weber") {
... on User {
pinnedRepositories(first: 6) {
edges {
node {
name
}
}
}
repositories(first: 100) {
edges {
node {
name
description
url
isFork
primaryLanguage {
name
}
stargazers {
totalCount
}
}
}
}
}
}
}

"#.to_string()
```
And when i try to use the request with graphql api its not working. But when I use it with:
https://developer.github.com/v4/explorer/
It is working.

I don't remember exactly but I get the response from github stated that it had some problem with parsing json.

**Comment by mgattozzi** [comment]
So I haven't touched this code in a while. I'm sure the API has changed a lot or something I don't know about. It's very much a thin wrapper around it. Are you using the new_raw() function? Also you don't need to call to_string() as the api does it for you. Could you also try without the raw string just to make sure it's not the problem? (I doubt it is)

**Comment by jaroslaw-weber** [comment]
yeah i passed it to newraw function. maybe because i dont use backslashes on new line like in the example?

**Comment by jaroslaw-weber** [comment]
ok so I found the problem and fix. i will send a pr soon.
the problem was that you send the body as json but you dont escape quotation marks and newlines.
i tested it locally and it was working (with my query i posted earlier)

**Comment by Dylan-DPC** [comment]
That's great :+1:

**Comment by mgattozzi** [comment]
Hey @jaroslaw-weber awesome thanks for figuring it out!
